api/api/model_endpoints_api.go

```diff
@@ -124,6 +125,9 @@ func (c *ModelEndpointsController) CreateModelEndpoint(r *http.Request, vars map
 		return InternalServerError(fmt.Sprintf("Error creating model endpoint: %v", err))
 	}
 
+	// trigger webhook call
+	_ = c.Webhook.TriggerModelEndpointEvent(ctx, webhook.OnModelEndpointCreated, endpoint)
```

Comment:
seems for all webhook trigger error is ignored. Is it expected? 
How about if we add log warning while the webhook error?

Reply:
Yes it's expected because as of now, the usage of the webhook doesn't modify the resource in Merlin, meaning if there's error happening the webhook call, we do not want to rollback/cancel the changes of the resource. 

And the log warning is inside the `TriggerModel`, I think it suffice?

Reply:
I've changed the way this interface work now. Please check it again :D There will only be a general function that can be called by all event, with option to setup the body, success callback, and error callback (by default the error will be default callback from the library). 

The error is now returned and processed, most of it will just log it with a `warning` level thought. If we do want to fail it or return the error to user, I believe we need a mechanism to rollback the creation/update of resource (because it will be weird if we simply return an error but actually the creation is success), perhaps later if we have a clear usecase of how we're going to fail/rollback this, we can update it again